openwisp_radius/migrations/0015_freeradius_allowed_hosts.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 3.0.7 on 2020-07-12 15:46
+
+import jsonfield.fields
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('openwisp_radius', '0014_radiustoken_freeradius_auth'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='organizationradiussettings',
+            name='freeradius_allowed_hosts',
+            field=models.TextField(
+                blank=True,
+                help_text='Comma separated list of IP addresses allowed to access freeradius API',
+                null=True,
+            ),
+        ),
+        migrations.AlterField(
+            model_name='organizationradiussettings',
+            name='sms_meta_data',
+            field=jsonfield.fields.JSONField(
```

Comment:
here we changed only the help_text right? Can you edit the previous migration instead of adding this `AlterField`? These kinds of small improvements make the code more easy to maintain in the long run. (Imagine this kind of change being done in every migration, it would make the whole migrations harder to follow).